app/helpers/application_helper.rb

```diff
@@ -1,5 +1,10 @@
-# frozen_string_literal: true
-
-# ApplicationHelper
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
```

Comment:
còn cách khác k e

Reply:
có thể dùng toán tử 3 ngôi, 
page_title? (page_title + " | " + base_title) : base_title

Reply:
Em gửi luôn trong pull tiếp theo luôn ạ

Reply:
page_title ? (page_title + " | " + base_title) : base_title